views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/recorder.js

```diff
@@ -200,8 +199,35 @@ define([
                         initAnalyser(stream);
 
                         setState(recorder, recorderStates.IDLE);
+                    })
+                    .catch(function(err) {
+                        alert(err);
```

Comment:
`alert` is not allowed, we need to pass the error to the testRunner to get handled properly.

Reply:
fixed